src/BinaryParsers/BinaryParsers.csproj

```diff
@@ -2,12 +2,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., build.netcore.props))\build.netcore.props" />
   <PropertyGroup>
-    <RootNamespace>Microsoft.CodeAnalysis.BinaryParsers</RootNamespace> 
+    <RootNamespace>Microsoft.CodeAnalysis.BinaryParsers</RootNamespace>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>

```

Comment:
Is there a reason we're now building both of these here?  This is a library, it should be consumable as .net standard from both .net 461 and .net core.

(Also--we have macros for these if we need to build both for some reason.)